doc/deployment/connect-to-cluster.md

```diff
@@ -0,0 +1,187 @@
+# Connect to a Pachyderm cluster
+
+After you deploy a Pachyderm cluster, you can continue to use
+the command-line interface, connect to the Pachyderm dashboard, or
+configure third-party applications to access your cluster programmatically.
+Often all you need to do is just continue to use the command-line
+interface to create repositories, pipelines, and upload your data.
+At other times, you might have multiple Pachyderm clusters deployed
+and need to switch between them to perform management operations.
+
+You do not need to configure anything specific to start using the
+Pachyderm CLI right after deployment. However, the Pachyderm
+dashboard and the S3 gateway require
+explicit port-forwarding or direct access through an externally
+exposed IP address and port.
+
+This section describes the various options available for you to connect
+to your Pachyderm cluster.
+
+## Connect to a Local Cluster
+
+If you are just exploring Pachyderm, use port-forwarding to
+access both `pachd` and the Pachyderm dashboard.
+
+By default, Pachyderm enables port-forwarding from `pachctl` to `pachd`.
+If you do not want to use port-forwarding for `pachctl` operations,
+configure a `pachd_address` as described in
+[Connect by using a Pachyderm context](#connect-by-using-a-pachyderm-context.html).
+
+To connect to the Pachyderm dashboard, you can either use port-forwarding,
+or the IP address of the virtual machine on which your Kubernetes cluster
+is running.
+
+The following example shows how to access a Pachyderm cluster
+that runs in `minikube`.
+
+To connect to a Pachyderm dashboard, complete the following steps:
+
+* To use port-forwarding, run:
+
+  ```bash
+  pachctl port-forward
+  ```
+
+* To use the IP address of the VM:`
+
+  1. Get the minikube IP address:
+
+     ```bash
+     $ minikube ip
+     ```
+
+  1. Point your browser to the following address:
+
+     ```bash
+     <minikube_ip>:30080
+     ```
+
+## Connect to a Cluster Deployed on a Cloud Platform
+
+Similarly to a local cluster deployment, in Pachyderm
```

Comment:
Suggest:

As in a local cluster deployment, 
a Pachyderm cluster deployed on a cloud platform has impllcit port-forwarding enabled.
This means that,
if you're connected to the cluster so that `kubectl` works,
`pachctl` can talk to `pachd` without doing anything .
Other services will still need explicit port forwarding.
For example, 
to access the Pachyderm dashboard, 
you need to run `pachctl port-forward`.